src/api/spec/controllers/search_controller_spec.rb

```diff
@@ -38,6 +39,21 @@
     subject! { get :package, params: { match: "@name='apacheX'" } }
 
     it_behaves_like 'find package'
+
+    context 'With offset' do
+      subject! { get :package, params: { offset: 'XXX', match: "@name='apacheX'" } }
+
+      it_behaves_like 'find package'
```

Comment:
I'm not sure what you are testing here? That `offset: 'XXX'` has no effect?

Reply:
yes, well i added this spec before changing the code.. and before the original code was doing some manual convertion: https://github.com/openSUSE/open-build-service/blob/master/src/api/app/controllers/search_controller.rb#L125, i can drop it now for sure.

Reply:
Yeah a simple `.to_i` should suffice...

Reply:
done